Source/UIScrollView+EmptyDataSet.m

```diff
@@ -740,17 +740,19 @@ - (instancetype)init
 
 - (void)didMoveToSuperview
 {
-    self.frame = self.superview.bounds;
-    
-    void(^fadeInBlock)(void) = ^{_contentView.alpha = 1.0;};
-    
-    if (self.fadeInOnDisplay) {
-        [UIView animateWithDuration:0.25
-                         animations:fadeInBlock
-                         completion:NULL];
-    }
-    else {
-        fadeInBlock();
+    [super didMoveToSuperview];
+    if (self.superview) {
```

Comment:
It turns out that when the view is `removed` from superview, this method will get called too. For that case, self.superview would be nil.